### PR TITLE
Add DicMusic.club site support

### PR DESCRIPTION
**Pull request by gbmf**
Add DicMusic.club site support.
DicMusic.club is a Chinese Gazelle music PT site named DIC(Dolphin is Coming).

### CODE REVIEW

**Comment by SavageCore** [comment]
Looks good, thanks for the PR. Can you remove the version bump? Then I will merge.

File: pth_yadg.user.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 // @name           RED YADG
 // @description    This script provides integration with online description generator YADG (http://yadg.cc) - Credit to Slack06
 // @license        https://github.com/SavageCore/yadg-pth-userscript/blob/master/LICENSE
-// @version        1.6.3
+// @version        1.6.4
```
Comment (SavageCore) [inline pth_yadg.user.js]: To remove

Comment (gbmf) [reply pth_yadg.user.js]: Have removed the version bump, Please merge, Thanks a lot !

**Comment by SavageCore** [comment]
@all-contributors please add @gbmf for DicMusic.club support

**Comment by SavageCore** [comment]
@all-contributors please add @gbmf for 💻

**Comment by SavageCore** [comment]
@all-contributors please add @gbmf for code

**Comment by SavageCore** [comment]
@all-contributors please add @gbmf for code

**Pull request merged**